src/test/app/Loan_test.cpp

```diff
@@ -7062,7 +7072,6 @@ class Loan_test : public beast::unit_test::suite
         testInvalidLoanDelete();
         testInvalidLoanManage();
         testInvalidLoanPay();
-        testInvalidLoanSet();
```

Comment:
Got a hitchhiking `return` here, and you might want to put `testInvalidLoanSet` back to it's original position.